asapdiscovery/docking/mcs.py

```diff
@@ -287,7 +287,7 @@ def rank_structures_rdkit(
             d.FinishDrawing()
             d.WriteDrawingText(f"{out_fn}_{search_ids[mol_idx]}_{i}.png")
 
-    Draw.MolToFile(exp_mol, f"{out_fn}.png")
+        Draw.MolToFile(exp_mol, f"{out_fn}.png")
```

Comment:
could we make this optional? I'm not sure what the `.png`s are used for and I end up just deleting them manually.

Reply:
as of [64ad66f](https://github.com/choderalab/covid-moonshot-ml/pull/93/commits/64ad66ffc4c5dc02167a3b3c09f731a4172c94b3) the script won't draw the molecules if you set `-n_draw 0`